backend/src/helpers/index.ts

```diff
@@ -0,0 +1,17 @@
+import { paramCase, Options } from 'change-case';
+import { Case } from '~/common/enums/case/case.enum';
+
+const caseTypeToFn: Record<Case, typeof paramCase> = {
+  'kebab': paramCase,
```

Comment:
You can use enum value to provide a key for object

Reply:
not sure that we need an enum. because we will now work it somehow let's use just an union